velox/dwio/parquet/tests/ParquetWriterTestBase.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <gtest/gtest.h>
+#include <string>
+#include "velox/common/base/Fs.h"
+#include "velox/dwio/common/FileSink.h"
+#include "velox/dwio/common/tests/utils/BatchMaker.h"
+#include "velox/dwio/common/tests/utils/DataFiles.h"
+#include "velox/dwio/parquet/writer/Writer.h"
+#include "velox/exec/tests/utils/TempDirectoryPath.h"
+#include "velox/vector/tests/utils/VectorMaker.h"
+
+namespace facebook::velox::parquet {
+
+class ParquetWriterTestBase : public testing::Test {
+ protected:
+  void SetUp() override {
+    dwio::common::LocalFileSink::registerFactory();
+    rootPool_ = memory::defaultMemoryManager().addRootPool("ParquetWriterTest");
+    leafPool_ = rootPool_->addLeafChild("SinkTest");
+  }
+
+  fs::path createTempPath() {
+    auto root = exec::test::TempDirectoryPath::create();
+    return fs::path(fmt::format("{}/test_file.txt", root->path));
+  }
+
+  std::vector<RowVectorPtr> createBatches(
+      const std::shared_ptr<const Type>& rowType,
+      uint64_t batchNum,
+      uint64_t capacity) {
+    std::vector<RowVectorPtr> batches;
```

Comment:
@xiaoxmeng We could not use fuzzer here, for it would generate a constant vector, which cannot be exported to Arrow yet. The fuzzer has no opt to disable the generation of constant vectors.
Could I use the current generation method in this PR, and file another PR to add the opt to disable constant vector generation and then update this part?